package.json

```diff
@@ -12,6 +12,9 @@
       "stage-0"
     ]
   },
+  "react-svg-icons": {
+    "assetPath": "src/img/global/svg-icons"
```

Comment:
@DaleMcGrew It *was* simple! the path just needed to be declared within a top-level entry for react-svg-icons instead of in devDependencies where I had it.